slack-api-client/src/main/java/com/slack/api/methods/request/team/TeamInfoRequest.java

```diff
@@ -13,4 +13,9 @@
      */
     private String token;
 
+    /**
+     * Id of the team to get info on, if omitted, will return information about the current team.
```

Comment:
We want to copy the latest version of the description in the API document as-is.
```suggestion
     * Team to get info on, if omitted, will return information about the current team. Will only return team that the authenticated token is allowed to see through external shared channels
```

Reply:
I know the description in the website can be changed in the future but in this case, the latter half should be helpful for developers.

Reply:
Updated, thanks!